src/main/java/com/osa/fiels/ReadTxtfile.java

```diff
@@ -0,0 +1,22 @@
+package com.osa.fiels;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class ReadTxtfile {
```

Comment:
Please removed any unncessary code, like main method that we do not need in this framewrok